common/internal_model/src/main/scala/uk/ac/wellcome/models/parse/FreeformDate.scala

```diff
@@ -0,0 +1,103 @@
+package uk.ac.wellcome.models.parse
+
+import java.time.LocalDate
+
+import uk.ac.wellcome.models.work.internal.InstantRange
+
+/**
+  *  Models freeform text representing some date or period
```

Comment:
We generally do not document our code like this, but it's a pretty good idea in any case 👍

Reply:
... though I would only do it if it's going to be consistent.